### PR TITLE
Fix Grid documentation

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Fix README of Grid component and change title of styleguide

#### Why?

Because the old `size` attribute was used instead of `colSpan`

### CODE REVIEW

**Comment by luca-rath** [comment]
The failing pipeline shouldn't be related to this pull request

File: styleguide.config.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ const javaScriptFileExists = (path, fileName) => {
 const babelConfig = JSON.parse(fs.readFileSync(path.resolve(__dirname, '.babelrc'))); // eslint-disable-line no-undef
 
 module.exports = { // eslint-disable-line
+    title: 'Sulu Styleguide',
```
Comment (luca-rath) [inline styleguide.config.js:36]: The default title until now was `Sulu Sulu Style Guide`

Comment (luca-rath) [reply styleguide.config.js]: Just renamed that to `Sulu Javascript Docs`

**Comment by danrot** [comment]
I know about the PHP error, that is "fine". But the circle build should not fail... Maybe you didn't rebase that PR against the latest changes?

**Comment by luca-rath** [comment]
@danrot rebased

**Pull request merged**